planet.css

```diff
@@ -1,151 +1,241 @@
+/* Regular theme starts */
+.normalTheme{
+	--main-bg-color: #ffffff;
+	--nav-font-color: #b51418;
+	--box-shadow-color: rgba(221, 220, 220, 0.911);
+	--nav-btn-bgcolor: #ffffff;
+	--close-btn-hover: #d85f61; 
+	--content-font: #000000;
+	--title-border: #a0a0a0d8;
+
+} 
+/* Regular Theme ends */
+
```

Comment:
Add the below code to avoid horizontal scroll.
html{
	overflow-x: hidden;
}

Reply:
The overflow was supposed to be there for future mobile views, if this gets improved over the next week, then some more changes can be done 😄